plugin/switch.vim

```diff
@@ -216,6 +216,18 @@ autocmd FileType scala let b:switch_definitions =
       \ [
       \   g:switch_builtins.scala_string,
       \ ]
+autocmd FileType gitrebase let b:switch_definitions =
+      \ [
+      \   [ 'pick', 'fixup', 'reword', 'edit', 'squash', 'exec', 'drop' ],
+      \   [ '\^p ', 'fixup' ],
+      \   [ '\^f ', 'reword ' ],
+      \   [ '\^r ', 'edit' ],
+      \   [ '\^e ', 'squash ' ],
+      \   [ '\^s ', 'exec ' ],
+      \   [ '\^x ', 'drop ' ],
+      \   [ '\^d ', 'pick ' ],
+      \ ]
```

Comment:
I think this ordering makes sense. I'd rather build it up like this, though:

``` vim
autocmd FileType gitrebase let b:switch_definitions =
       \ [
       \   [ 'pick', 'fixup', 'reword', 'edit', 'squash', 'exec', 'drop' ],
       \   { '^p ': 'fixup ' },
       \   { '^f ': 'reword ' },
       \   { '^r ': 'edit ' },
       \   { '^e ': 'squash ' },
       \   { '^s ': 'exec ' },
       \   { '^x ': 'drop ' },
       \   { '^d ': 'pick ' },
       \ ]
```

The reason is that the list form creates "looped" mappings -- from the first word to the second, and so on, and then from the last to the first. So, `p` would go to `fixup`, but `fixup` would also get a mapping to go to `p`. Of course, in this case, `fixup` gets caught early by the previous definition, but I think there's just no sense in creating the mapping anyway -- a `{ 'pattern': 'replacement' }` will do the same job. What do you think?